students/preactive/Lesson2/grid-alt.py

```diff
@@ -0,0 +1,24 @@
+def formation(gridSize, gridMulti):
+    plusEl = "+ "
```

Comment:
Python conventions is to use 'snake case' for all variable, function and parameter names. So these should be `grid_size`, `grid_multi`, `m_el` etc.

Reply:
Adjusted variable names to reflect snake case and pep8